src/frontend/js/order/orderApi.js

```diff
@@ -1,4 +1,29 @@
+/* globals API_URL */
 import { fetchEntitiesFactory, fetchEntityFactory } from '../app/entities/fetchEntities';
 
 export const fetchOrders = fetchEntitiesFactory('orders');
 export const fetchOrder = fetchEntityFactory('orders');
+
+export const fetchNewOrder = (products, jwt) => fetch(`${API_URL}/orders`, {
```

Comment:
The `credentials: 'include'` is missing, then the cookie is not sended to the API, and it throws an error.